server/clickhouse/schema.sql

```diff
@@ -0,0 +1,52 @@
+CREATE TABLE IF NOT EXISTS tool_logs
+(
+    ts              DateTime64(3, 'UTC'),
+    organization_id LowCardinality(String),
+    project_id      LowCardinality(String),
+    deployment_id   LowCardinality(String),
+    tool_type       Enum8('http'=1,'function'=2),
+    tool_id         LowCardinality(String),
+
+    trace_id        FixedString(32),
+    span_id         FixedString(16),     -- should we use UUID here?
+
+    level           Enum8('TRACE'=1,'DEBUG'=2,'INFO'=3,'WARN'=4,'ERROR'=5),
+    message         String CODEC (ZSTD), -- should this be capped?
```

Comment:
we'll cap in the application layer if needed. we control the messages that go into this table anyway.